apps/webapp/web/static/js/docs/index.js

```diff
@@ -5,18 +5,33 @@ import material from 'material-design-lite/material.css';
 import styles from 'docs/styles.css';
 import TableOfContents from 'docs/TableOfContents';
 import DocPage from 'docs/DocPage';
+import fetch from 'isomorphic-fetch';
 
 class Index extends React.Component {
 
   constructor(props) {
     super();
+    this.state = {docs: []};
+  }
+
+  componentDidMount() {
+    fetch(window.location.origin + '/admin/docs/list_of_docs')
```

Comment:
Please add above `fetch` 
```javascript
// TODO: change window.location.origin to a URL variable rendered from the server. 
```
The problem is that if this Javascript is hosted in a CDN such as CloudFront, then it will try to fetch the **list_of_docs** from there instead of the Phoenix app server.

PS: As I mentioned in our Scrum, you could also create a field and resolver to get the **list_of_docs** from the GraphQL schema and avoid having to fetch manually.  The query would be simply something like:
```graphql
query {
  list_of_docs 
}
```